testcases/bestest_hydronic/models/generate_data.py

```diff
@@ -153,5 +153,54 @@
 # Add CO2 limits
 df['UpperCO2[1]'] = 894.
 
+def hold_step_values_in_event_times(df, var, event_freq):
+    ''' This method goes through the index of a data frame to inspect
+    when step changes take place for a variable and makes sure that
```

Comment:
Maybe just clarify as "positive" step changes?

Reply:
Yep! clarified in https://github.com/ibpsa/project1-boptest/pull/311/commits/ed5d3b8e41f2e14d4f0a23e731e27e5c60a845c0.

Reply:
Ok thanks.  Will merge after checks pass one last time.